### DISCUSSION TITLE
Fingerprintreader defekt?

### DISCUSSION THREAD

**Discussion by Maggus574** [frickelzeugs/FingerprintDoorbell#78] [Q&A]
Hey!

Spannendes Projekt! Habe mir auch einen R503 Fingerprintreader zugelegt und einen ESP32 mit dem FingerprintDoorbell - Projekt geflashed. Soweit hat das anscheinend auch geklappt und ich kann über die IP - Adresse darauf zugreifen, Fingerprints anlegen, und sehe auch das die Fingerabdrücke erkannt werden. Über MQTT werden die Aktionen auch veröffentlicht. Muss nur noch schauen, wie ich das in HomeAssistant als Trigger nehmen kann... ;-)

Was mich aber wundert: nach dem Start des ESP blinkt die LED vom Reader nur rot. Erreichbar ist er aber. Und nach dem ersten Scan leuchtet die LED dann dauerhaft lila. Aber blau, wie in der Doku beschrieben, war sie noch nie.

Hatte das schon mal Jemand? Oder ist mein Fingerprint - Reader vielleicht kaputt?

Bis denn...

Bye, Markus

**Comment by Maggus574** [comment]
Vielleicht noch ein Hinweis: alle zwei Sekunden kommt "No Match Found (Code 2)". Sollte das nicht nur dann kommen, wenn ein noch nicht registrierter Fingerabdruck gescannt wird? Und schaltet die LED vielleicht deshalb nicht auf blau um?

**Reply by PinkyFooFoo** [reply]
Das klingt für mich nach einem defekten Fingerprint-Sensor. Eventuell ist auch die Luftfeuchtigkeit zu hoch und er erkennt "regen" bzw. feuchte Luft als Finger. Das ist aber so wie du es beschreibst eher unwahrscheinlich. Das "No Match Found" klingt danach als würde er laufend Finger erkennen wo keine da sind. Wenn du nichts "umkonfiguriert" hast würde ich meinen dass du mit dem investment in einen neuen Fingerprint-Sensor den besten Weg zum debuggen gehst.

**Reply by Maggus574** [reply]
Ich befürchte das Du Recht hast. Dazu würde auch passen, daß die LED dauerhaft lila leuchtet. Habe mal einen zweiten Sensor bestellt. Bin gespannt wie der sich verhält.

**Reply by Maggus574** [reply]
Ach menno...

So schnell der neue Reader auch angekommen ist, so ernüchternd war bzw. ist das Ergebnis. :-( Er verhält sich exakt, wie der vorherige Reader und schickt auch alle zwei Sekunden eine "No Match found (Code 2)" - Nachricht raus. Trotzdem kann ich Fingerabdrücke speichern und er erkennt sie auch.

Ich versteh´s nicht... Die Belegung bin ich etliche Male durchgegangen und sollte eigentlich passen. Einzig habe ich statt einem grünen Kabel ein eher lila oder braunes Kabel. Oder hat das Ding damit eine komplett andere Belegung? Aber dann dürfte er ja eher nichts Speichern oder Lesen können. Was triggert das Ding, das er permanent ohne Finger liest?

**Comment by Maggus574** [comment]
Ok, ich bin einen Schritt weiter: ich habe spaßeshalber mal die Einstellung "fingerprintDoorbell/ignoreTouchRing" auf "on" gesetzt, und daraufhin geht die Status - LED auf blau. Und auch die Scans alle zwei Sekunden haben damit aufgehört, und nur gelesene Fingerabdrücke werden erkannt und per MQTT gepublished. Leider ist die Einstellung default nach einem Reboot wieder "off" schade, daß das nicht direkt in den Einstellungen mitgegeben werden kann. Aber da lässt sich sicher was über die Automation in HomeAssistant machen.

Jetzt muss ich mal weiter schauen, wie ich die MQTT - Topics als Auslöser in HomeAssistant nutzen kann. So direkt als Dienst auf ein MQTT - Topic zu hören scheint leider alleine noch nicht zu langen... Aber das ist ein anderes Thema! ;-)
